Packs/Slack/Integrations/SlackV3/SlackV3.yml

```
@@ -503,13 +503,33 @@ script:
       description: Set this argument to specify how many results to return.
     description: Retrieves replies to specific messages, regardless of whether it's from a public or private channel, direct message, or otherwise.
     name: slack-get-conversation-replies
+  - arguments:
+    - default: true
+      description: 'The message to send to the channel. Use issue fields as variables to enrich the text. Example: “${issue.details}”.'
+      name: message
+      prettyname: Message
+      required: true
+    - description: The name of the Slack channel to which to send the message.
+      name: channel
+      prettyname: Channel
+      required: true
+    description: Sends a message to a user, group, or channel.
+    name: slack-send-notification-quick-action
+    prettyname: Send Slack Message
+    hidden: true
+    quickaction: true
+    outputs:
+      - contextPath: Slack.Thread.ID
+        description: The Slack thread ID.
+        type: String
   dockerimage: demisto/slackv3:1.0.0.117556
```

```suggestion
  dockerimage: demisto/slackv3:1.0.0.2155785
```

https://github.com/demisto/content/actions/runs/14064337229/job/39382853774?pr=38979#step:8:2593